mods/common/languages/en.ftl

```diff
@@ -717,3 +717,24 @@ options-time-limit =
         }
 
 notification-time-limit-expired = Time limit has expired.
+
+## EditorActorBrush
+notification-added-actor = Added { $name } ({ $id })
+## EditorCopyPasteBrush
+notification-copied-tiles = Copied { $amount } tiles
+## EditorDefaultBrush
+notification-removed-actor = Removed { $name } ({ $id })
+notification-removed-resource = Removed { $type }
+## EditorResourceBrush
+notification-added-resource =
+    { $amount ->
+       [one] Added { $amount } cell of { $type }
+      *[other] Added { $amount } cells of { $type }
+    }
+## EditorTileBrush
+notification-added-tile = Added tile { $id }
+notification-filled-tile = Filled with tile { $id }
+## EditorActionManager
+notification-opened = Opened
+## ActorEditLogic
+notification-edited-actor = Edited { $name } ({ $id })
```

Comment:
the current file style is to have whitespaces before comments

Reply:
They are all the same (editor brushes).

Reply:
That's the current style though. If we are to change style we should apply it to all en.ftl files

Reply:
Adding the white spaces is not too bad.